lalrpop/src/normalize/prevalidate/mod.rs

```diff
@@ -185,6 +185,8 @@ impl<'grammar> Validator<'grammar> {
             SymbolKind::Nonterminal(_) => {
                 /* see resolve */
             }
+            SymbolKind::Error => {
+            }
```

Comment:
If we were going to give a nicer error when combining `error` and `recursive-ascent`, this *might* be an ok place to do it. I forget if we know what kind of backend we are using at this point or what.